mantle/platform/api/gcloud/compute.go

```diff
@@ -120,14 +162,27 @@ func (a *API) mkinstance(userdata, name string, keys []*agent.Key, useServiceAcc
 			OnHostMaintenance: "TERMINATE",
 		}
 	}
-	return instance
-
+	// attach aditional disk
+	if len(opts.AdditionalDisks) > 0 {
+		for _, spec := range opts.AdditionalDisks {
+			plog.Debugf("Parsing disk spec %q\n", spec)
+			disk, err := ParseDiskSpec(spec, a.options.Zone)
+			if err != nil {
+				return nil, fmt.Errorf("failed to parse spec: %s\n", spec)
```

Comment:
We can wrap the error and drop the newline. E.g.

```suggestion
				return nil, fmt.Errorf("failed to parse spec %s: %w", spec, err)
```